app/code/AKisilenko/ModuleLesson6/view/frontend/email/ask_question_to_customer.html

```diff
@@ -0,0 +1,45 @@
+<!--@subject Email Subject @-->
+<!--@vars
+{"store url=\"\"":"Store Url",
```

Comment:
Are these two templates different? If they are the same, you can use one of them, no need to create the copy.
Just answer, no need to fix. Or you can add some changes to customer's template (imagine it should have some fancy design)

Reply:
The idea was to create 2 different templates, but I didn’t do this until the end, because I have a lot of work and I forgot ... I can make two different templates, if necessary. But most importantly, I understood how it works :)